qemu/tests/spapr_vty_multi_backends.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import os
+import re
+
+from virttest import error_context
+from virttest import remote
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify the login guest with multi backends spapr-vty:
+    1) Boot guest with multi spapr-vty with backend
+    2) Modify the kernel cfg file to specify the backend
+    3) For pty and file backend:
+      3.1) Open and close chardev
+    4) For unix_socket and tcp_socket:
+      4.1) Login guest
+      4.2) Create files inside guest
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    prompt = params.get("shell_prompt")
+    create_file = params["create_file"]
+    vm = env.get_vm(params["main_vm"])
+    vm.wait_for_login()
+    for serial_id in params.objects("serials"):
+        if serial_id != "vs1":
+            hvc_id = int(serial_id.replace('vs', '')) - 1
+            kernel_params = "console=hvc%s,115200" % hvc_id
+            utils_test.update_boot_option(vm, args_added=kernel_params)
```

Comment:
Is it necessary to remove the last parameter added here?

Reply:
I don't think it necessary, because turning on "image_snapshot = yes" will not affect the loop case.

Reply:
Oh, ignore this. It will configure the same argument.